src/esp_loader.ts

```diff
@@ -670,15 +669,15 @@ export class ESPLoader extends EventTarget {
     let flashWriteSize = this.getFlashWriteSize();
     if (
       !this.IS_STUB &&
-      [CHIP_FAMILY_ESP32, CHIP_FAMILY_ESP32S2].includes(this.chipFamily)
+      [CHIP_FAMILY_ESP32, CHIP_FAMILY_ESP32S2, CHIP_FAMILY_ESP32S3, CHIP_FAMILY_ESP32C3].includes(this.chipFamily)
     ) {
       await this.checkCommand(ESP_SPI_ATTACH, new Array(8).fill(0));
     }
-    if (this.chipFamily == CHIP_FAMILY_ESP32) {
+    //if (this.chipFamily == CHIP_FAMILY_ESP32) {
```

Comment:
Remove this code if it's no longer needed.

Reply:
done